src/pages/upload/index.tsx

```diff
@@ -295,6 +305,7 @@ const Upload: React.FC<
             <label className="block text-sm font-medium text-gray-700">
               Video Files
             </label>
+            <VideoUploadAlert />
```

Comment:
![CleanShot 2022-08-23 at 16 42 37@2x](https://user-images.githubusercontent.com/694063/186271873-6933f21b-29a8-41c4-b5aa-5ef146fe93a5.png)

Display some information to avoid failed uploads (for now, still root-causing this).